Fix build on Darwin with GCC. May need some adjustment for LLVM/Clang,
but I couldn't find the docs for possible compiler name types (for now).

It'll likely need a hack in the ebuild anyway.
--- a/setup.py
+++ b/setup.py
@@ -54,9 +54,6 @@ class BuildExt(build_ext):
         for ext in self.extensions:
             ext.include_dirs.insert(0, cppy.get_include())
             ext.extra_compile_args = opts
-            if sys.platform == 'darwin':
-                ext.extra_compile_args += ['-stdlib=libc++']
-                ext.extra_link_args += ['-stdlib=libc++']
             if (ct == 'msvc' and os.environ.get('KIWI_DISABLE_FH4')):
                 # Disable FH4 Exception Handling implementation so that we don't
                 # require VCRUNTIME140_1.dll. For more details, see:
